code/API_definitions/qos-profiles.yaml

```
@@ -47,33 +47,23 @@ tags:
 
 paths:
   /qos-profiles:
-    get:
+    post:
       tags:
         - QoS Profiles
-      summary: "Get All QoS Profiles"
+      summary: Get QoS profiles avaialable for a given device
       description: |
-        Returns all QoS Profiles that match the given criteria, or all profiles if no criteria is specified.
-
-        The access token may be either a 2-legged or 3-legged access token. If the access token is 3-legged, all returned QoS Profiles must be available to all end users associated with the access token.
-
-      operationId: getQosProfiles
-      security:
-        - openId:
-            - qos-profiles:qos-profiles:read
+        Returns all QoS Profiles that match the given criteria.
+        The access token may be either a 2-legged or 3-legged access token. If the access token is 3-legged, all returned QoS Profiles will be available to all end users associated with the access token.
```

Alignment of formatting with quality-on-demand

```suggestion
        **NOTES:**
        - The access token may be either a 2-legged or 3-legged access token. 
        - If the access token is 3-legged, all returned QoS Profiles will be available to all end users associated with the access token.
```